packages/storage-adapter-indexeddb/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ciscospark/storage-adapter-indexeddb",
+  "version": "0.7.0-alpha",
+  "description": "",
+  "main": "dist/index.js",
+  "devMain": "src/index.js",
+  "author": "Matt Norris <matthew.g.norris@gmail.com>",
+  "license": "(Apache-2.0)",
+  "repository": "https://github.com/ciscospark/spark-js-sdk/tree/master/packages/storage-adapter-indexeddb",
+  "dependencies": {
+    "@ciscospark/common": "^0.7.0-alpha",
+    "@ciscospark/spark-core": "^0.7.0-alpha",
+    "babel-runtime": "^6.3.19",
+    "localforage": "^1.4.2",
+    "lodash": "^4.5.1"
+  },
+  "devDependencies": {
+    "@ciscospark/storage-adapter-spec": "^0.7.0-alpha",
+    "@ciscospark/test-helper-chai": "^0.7.0-alpha",
```

Comment:
I think most of the test helper plugins can be eliminated since you're using the `storage-adapter-spec`.